src/rentalCarCost.test.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+const total = require('./rentalCarCost');
+
+test(`Do not add any discount for less than 3 days`, () => {
+  expect(total(2))
+    .toBe(80);
+});
+
+test('Should add the discount for 3 and more days of rent', () => {
+  expect(total(4))
+    .toBe(140);
+});
+
+test('Should add an additional discount for 7 and more days of rent', () => {
```

Comment:
When testing the code, we typically split values into equivalence classes and test their boundaries. 

In this case, equivalence classes would be:
[0 - 2], 
[3 - 6], 
[7 - inf)

you typically want to test boundary values: 2, 3, 6, 7 because that's where errors typically happen.